server/routes/induction/common/qualificationLevelController.ts

```diff
@@ -14,10 +14,8 @@ export default abstract class QualificationLevelController extends InductionCont
     res: Response,
     next: NextFunction,
   ): Promise<void> => {
-    const { pageFlowHistory, prisonerSummary } = req.session
-    if (!pageFlowHistory) {
```

Comment:
This was just wrong! and didnt work for our new simplified question set journey
I think it was there to support a feature of the previous short/long journeys, but was getting in the way of what we want to do now. 
Removing it broke no tests, so I can only assume we removed the relevant tests for this when we simplified the question set but didnt remove this part of the implementation